package.json

```diff
@@ -17,26 +22,31 @@
   },
   "dependencies": {
     "dot": "~1.0.3",
-    "svgo": "~0.5.1",
-    "canvas": "~1.1.2",
+    "svgo": "~0.6.1",
+    "canvas": "~1.3.5",
     "phantomjs": "~1.9.2-6",
-    "es6-promise": "~2.1.0",
-    "request": "~2.55.0",
-    "redis": "~1.0.0",
-    "camp": "~15.5.5",
-    "semver": "~4.3.3",
-    "bower": "~1.4.1",
-    "promise": "~7.0.0"
+    "es6-promise": "~3.0.2",
+    "request": "~2.67.0",
+    "redis": "~2.4.2",
+    "camp": "~15.10.11",
+    "semver": "~5.1.0",
+    "bower": "~1.7.1",
+    "promise": "~7.1.1"
   },
   "devDependencies": {
     "ass": "~0.0.6",
-    "should": "~3.0.0",
-    "mocha": "~1.14.0"
+    "coveralls": "^2.11.6",
+    "istanbul": "^0.4.1",
```

Comment:
As mentioned, coverage is already part of the test. coveralls and istanbul are not necessary.